corehq/apps/reports/templates/reports/reportdata/bootstrap3/case_data.html

```diff
@@ -36,7 +36,7 @@
                   {% blocktrans %}
                     This case uses a deprecated case type.
                     See the
-                    <a target="_blank" href="https://confluence.dimagi.com/display/commcarepublic/Deprecating+Case+Types+and+Case+Properties+using+Data+Dictionary">help documentation</a>
+                    <a target="_blank" href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143944977/Data+Dictionary">help documentation</a>
```

Comment:
More specific: https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143944977/Data+Dictionary#Deprecating-%26-Delete-Case-Types-and-Case-Properties